### ISSUE TITLE
Button customization is missing

### ISSUE THREAD

**Issue by svilen1** [googlemaps/android-maps-compose#35]
Hello, I have an issue with customization of mylocation button and positioning it in specific area, in general there is no option to customize your button with new icon and so on.

**Comment by jpoehnelt** [comment]
@svilen1 Thank you for opening this issue. 🙏
Please check out these other resources that might be applicable:

- [Support Console](https://cloud.google.com/support/) - If you have a support contract.
- [Discord](https://discord.gg/9UpV862xZ9) - chat with other developers
- [StackOverflow](http://stackoverflow.com/questions/tagged/google-maps) - use the `google-maps` tag
- [Google Maps Platform issue trackers](https://developers.google.com/maps/support/#issue_tracker)


_This is an automated message, feel free to ignore._

**Comment by jpoehnelt** [comment]
@svilen1 Please take a moment to fill out [this short survey](https://forms.gle/wYuzN6UibnFj7ZL29). Thank you!

_This is an automated message, feel free to ignore._

**Comment by svilen1** [comment]
Have you worked on this issue?

**Comment by LeoBeliakov** [comment]
What about this issue?

**Comment by jmiecz** [comment]
Will support be added for this? Curious why it defaults top right when on Google Maps app is bottom right.

**Comment by pretRossB** [comment]
Support for this needs to be added as the existing UI and positioning is poor. I just want to have a custom fab bottom right of the screen but it doesn't support this. I'll have to roll my own location listening and UI on the map to do this instead of just setting a custom composable as the button

**Comment by kikoso** [comment]
Hello. Did you take a look at the [CustomControlActivity demo](https://github.com/googlemaps/android-maps-compose/blob/main/maps-app/src/main/java/com/google/maps/android/compose/CustomControlsActivity.kt#L104)? There is an example showcasing how to do precisely this.

**Comment by pretRossB** [comment]
> Hello. Did you take a look at the [CustomControlActivity demo](https://github.com/googlemaps/android-maps-compose/blob/main/maps-app/src/main/java/com/google/maps/android/compose/CustomControlsActivity.kt#L104)? There is an example showcasing how to do precisely this.

Yes and this is not a custom location button. This is just a button that displays a toast. I would expected it to be a button that when clicked on centers the camera on the current device location.
